lib/rspec/mocks/any_instance/expect_chain_chain.rb

```diff
@@ -24,6 +24,7 @@ def create_message_expectation_on(instance)
 
         def invocation_order
           @invocation_order ||= {
+            :with => [nil],
             :and_return => [nil],
             :and_raise => [nil],
             :and_yield => [nil]
```

Comment:
I've pushed up a refactor, see what you think, it will at least surface the true error rather than showing the `include?` one.